tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/jms/client/TemporaryDestinationTest.java

```diff
@@ -227,45 +228,6 @@ public void testForTempQueueCleanerUpperLeak() throws Exception {
       }
    }
 
-   @Test
```

Comment:
These tests are still valid and shouldn't be removed. They are testing the server-side accumulation of address info. The changes in this PR are for the client-side.